src/vtrasterizer/DecorationRenderer.cpp

```diff
@@ -218,7 +218,7 @@ auto DecorationRenderer::createTileData(Decorator decoration, atlas::TileLocatio
             auto const x0 = 0u;
             auto const x1 = unbox<unsigned>(width) / 2;
             auto block = blockElement(ImageSize { width, height });
-            return create(block.downsampledSize(), [&]() -> atlas::Buffer {
+            return create(block.getdownsampledSize(), [&]() -> atlas::Buffer {
```

Comment:
```suggestion
            return create(block.getDownsampledSize(), [&]() -> atlas::Buffer {
```
I think that's more readable if it starts capital lettered after the `get` prefix, what do you think?

Reply:
Changed